### DISCUSSION TITLE
Support for run position

### DISCUSSION THREAD

**Discussion by BobPritchett** [dolanmiu/docx#1851] [Ideas / Requests]
Is there a way to adjust run position from baseline with the position attribute? (It looks like it's missing as an option [here](https://github.com/dolanmiu/docx/blob/49b4ca67e0514653dc333ccf2a5f8ac4272165ac/src/file/paragraph/run/properties.ts), but I'm not sure I know all the right places to look.)

https://learn.microsoft.com/en-us/dotnet/api/documentformat.openxml.wordprocessing.position?view=openxml-2.8.1
<img width="283" alt="image" src="https://user-images.githubusercontent.com/26860966/209417180-dffa5fce-5908-4837-9f17-7779f826dde3.png">

**Comment by dolanmiu** [comment]
Thanks for this

This has been resolved in:

https://github.com/dolanmiu/docx/pull/1866
